AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -123,88 +124,103 @@ class TransactionConfigurator {
                     gasLimit: GasLimitConfiguration.maxGasLimit,
                     data: transaction.data ?? configuration.data
             )
-            completion(.success(()))
-        case .ERC20Token:
-            session.web3.request(request: ContractERC20Transfer(amount: transaction.value, address: transaction.to!.description)) { [unowned self] result in
-                switch result {
-                case .success(let res):
-                    let data = Data(hex: res.drop0x)
-                    self.configuration = TransactionConfiguration(
-                            gasPrice: self.calculatedGasPrice,
-                            gasLimit: GasLimitConfiguration.maxGasLimit,
-                            data: data
-                    )
-                    completion(.success(()))
-                case .failure(let error):
-                    completion(.failure(error))
-                }
+            return completion(.success(()))
+        case .ERC20Token(_):
```

Comment:
Some of these values should be read from `ContractERC721Transfer`, `ContractERC875Transfer `, `ContractERC20Transfer`.